res/values/strings_tl.arb

```diff
@@ -197,10 +198,12 @@
   "connection_sync": "Koneksyon at pag-sync",
   "connectWalletPrompt": "Ikonekta ang iyong wallet sa WalletConnect upang gumawa ng mga transaksyon",
   "contact": "Contact",
+  "contact no longer has an address assigne": "Ang contact na ito ay wala nang isang address na itinalaga. Mangyaring pumili ng ibang contact o mano -mano ang isang address. (Pera :)",
```

Comment:
wouldn't this break something?
it's in most translation files but not all of them